.github/workflows/lint.yml

```diff
@@ -73,8 +74,7 @@ jobs:
           cl/api/routers.py
           cl/api/tests.py
           cl/api/views.py
-          cl/opinion_page/views.py
```

Comment:
This is just general lack of knowledge on lint, but is removing opinion-page/views correct?

Reply:
Oh weird. Good catch. Fixed.